### DISCUSSION TITLE
Equivalent of anyio.from_thread.run_sync() ?

### DISCUSSION THREAD

**Discussion by dorinclisu** [fastapi/asyncer#145] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the Asyncer documentation, with the integrated search.
- [X] I already searched in Google "How to X in Asyncer" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to Asyncer but to [AnyIO](https://github.com/agronholm/anyio).
- [X] I already checked if it is not related to Asyncer but to [Trio](https://github.com/python-trio/trio).

### Commit to Help

- [X] I commit to help with one of those options 👆

### asyncer Version
0.0.5

### Description

Equivalency:
- `anyio.to_thread.run_sync` -> `asyncify`
- `anyio.from_thread.run` -> `syncify`
- `anyio.from_thread.run_sync` -> ?

https://anyio.readthedocs.io/en/latest/threads.html
Does `syncify` magically infer the kind of function and it calls the appropriate anyio method?

**Comment by tiangolo** [comment, marked as answer]
There is currently no equivalent to `anyio.from_thread.run_sync`, but you can call it directly if you need it.

I would wait for a use case that needs it before considering adding it. :nerd_face:
